nix/sources.nix

```diff
@@ -0,0 +1,174 @@
+# This file has been generated by Niv.
+
+let
+
+  #
+  # The fetchers. fetch_<type> fetches specs of type <type>.
+  #
+
+  fetch_file = pkgs: name: spec:
+    let
+      name' = sanitizeName name + "-src";
+    in
+      if spec.builtin or true then
```

Comment:
`or true`? @steveeJ

Reply:
i want to preface this by pointing out that this file is autogenerated so i didn't come up with this logic. 

consider this example:

```bash
$ nix repl
Welcome to Nix 2.5.0pre20211206_d1aaa7e. Type :? for help.

nix-repl> specWithBuiltinFalse = { builtin = false; }

nix-repl> specWithBuiltinTrue = { builtin = true; }

nix-repl> specWithoutBuiltin = { }

nix-repl>

nix-repl> specWithBuiltinFalse.builtin or true
false

nix-repl> specWithBuiltinTrue.builtin or true
true

nix-repl> specWithoutBuiltin.builtin or true
true
```

the `<expression> or <alternative expression>` syntax in Nix is not a logical OR. it's a form of handling an exception that might occur in the expression by providing an alternative expression in the exception case.

Reply:
@steveeJ kk